aws/codegen-aws-sdk/src/test/kotlin/software/amazon/smithy/rustsdk/RetryPartitionTest.kt

```diff
@@ -139,7 +139,7 @@ class RetryPartitionTest {
                             ) -> Result<(), #{BoxError}> {
                                 self.called.fetch_add(1, Ordering::Relaxed);
                                 let token_bucket = cfg.load::<#{TokenBucket}>().unwrap();
-                                let expected = format!("permits: {}", tokio::sync::Semaphore::MAX_PERMITS);
+                                let expected = format!("permits: 500000000");
```

Comment:
Should we expose this as a public constant `MAX_TOKENS`/`MAX_PERMITS` so we don't have to have magic numbers in tests